distributed/scheduler.py

```diff
@@ -3078,7 +3091,7 @@ def __init__(
         resources = {}
         aliases = {}
 
-        self._task_state_collections = [unrunnable]
```

Comment:
This construction feels a bit off and is a relict of Cython. I will come back to this again once we removed the SchedulerState